tests/unit_tests/test_morphology_utils.py

```diff
@@ -0,0 +1,75 @@
+"""Tests for EModelMetadata methods."""
+
+"""
+Copyright 2024, EPFL/Blue Brain Project
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import numpy
+import pytest
+
+from bluepyemodel.model.morphology_utils import get_apical_length
+from bluepyemodel.model.morphology_utils import get_apical_point_soma_distance
+from bluepyemodel.model.morphology_utils import get_basal_and_apical_lengths
+from bluepyemodel.model.morphology_utils import get_hotspot_location
+
+from tests.utils import DATA
+
+
+@pytest.fixture
+def morph_path():
+    return DATA / "morphology" / "C060114A5.asc"
+
+
+def test_get_apical_point_soma_distance(morph_path):
+    """Test get_apical_point_soma_distance function."""
+    soma_dist = get_apical_point_soma_distance(morph_path)
+    numpy.testing.assert_allclose(soma_dist, 624.8454)
+
+
+def test_get_apical_length(morph_path):
+    """Test get_apical_length function."""
+    apical_length =  get_apical_length(morph_path)
```

Comment:
Stupid question: why is it called `get_apical_length` while it is a radial distance?

Reply:
I will rename it get_apical_radial_distance, it would make more sense. To me, the concepts look similar, so I did not put much thinking into the naming

Reply:
Ok :)
Usually we use either `radial distance` or `path length` to be more specific (and sometimes `path distance` but it's usually to speak about a path lengths between 2 arbitrary points on the neurite while the path length is computed from the origin of the neurite).